.github/workflows/stalled.yml

```diff
@@ -0,0 +1,26 @@
+name: Close Stalled PRs
```

Comment:
Rename this to something like "Label Stalled PRs" or similar

Reply:
Thanks @andrross! Made the requested changes :)